### PR TITLE
Docstrings for `set_aog_theme!` and `aog_theme`

### PR DESCRIPTION
**Pull request by KronosTheLate**
What it says on the box.

### CODE REVIEW

File: src/theme.jl [hunk 1]
```diff
@@ -23,6 +23,12 @@ opensans(weight) = joinpath(font_folder, "OpenSans-$(weight).ttf")
 # Crameri, Fabio, Grace E. Shephard, and Philip J. Heron. "The misuse of colour in science communication." Nature communications 11.1 (2020): 1-10.
 # https://www.nature.com/articles/s41467-020-19160-7
 
+"""
+    aog_theme(; fonts=[firasans("Medium"), firasans("Light")])
+
+Returns a `NamedTuple` of theme settings. Intended for internal use.
```
Comment (piever) [inline src/theme.jl:29]:
```suggestion
Return a `NamedTuple` of theme settings. Intended for internal use.
```

File: src/theme.jl [hunk 2]
```diff
@@ -120,6 +126,29 @@ function aog_theme(; fonts=[firasans("Medium"), firasans("Light")])
     )
 end
 
+"""
+    set_aog_theme!(; kwargs...)
+
+Set the current theme to a predefined and opinionated theme,
+as defined by the internal function `aog_theme` (inernal means not
+exported).
```
Comment (piever) [inline src/theme.jl:134]:
```suggestion
as defined by the unexported internal function [`AlgebraOfGraphics.aog_theme`](@ref).
```

File: src/theme.jl [hunk 3]
```diff
@@ -120,6 +126,29 @@ function aog_theme(; fonts=[firasans("Medium"), firasans("Light")])
...
+
+To tweak the predefined theme, use the function `update_theme!`
+The change e.g. default fontsize, title, and markersize,
+see the example below.
```
Comment (piever) [inline src/theme.jl:138]:
```suggestion
To tweak the predefined theme, use the function `Makie.update_theme!`.
See the example below on how to change, e.g., default fontsize, title, and markersize.
```

File: src/theme.jl [hunk 4]
```diff
@@ -23,6 +23,12 @@ opensans(weight) = joinpath(font_folder, "OpenSans-$(weight).ttf")
 # Crameri, Fabio, Grace E. Shephard, and Philip J. Heron. "The misuse of colour in science communication." Nature communications 11.1 (2020): 1-10.
 # https://www.nature.com/articles/s41467-020-19160-7
 
+"""
+    aog_theme(; fonts=[firasans("Medium"), firasans("Light")])
+
+Returns a `NamedTuple` of theme settings. Intended for internal use.
+The provided functionality is exposed to the user by the function `set_aog_theme!`.
```
Comment (piever) [inline src/theme.jl:30]:
```suggestion
The provided functionality is exposed to the user by the function [`set_aog_theme!`](@ref).
```

File: src/theme.jl [hunk 5]
```diff
@@ -120,6 +126,29 @@ function aog_theme(; fonts=[firasans("Medium"), firasans("Light")])
...
+
+Set the current theme to a predefined and opinionated theme,
+as defined by the internal function `aog_theme` (inernal means not
+exported).
+
+To tweak the predefined theme, use the function `update_theme!`
+The change e.g. default fontsize, title, and markersize,
+see the example below.
+
+For more information about setting themes, see the `theming`
```
Comment (piever) [inline src/theme.jl:140]:
```suggestion
For more information about setting themes, see the `Theming`
```

File: src/theme.jl [hunk 6]
```diff
@@ -120,6 +126,29 @@ function aog_theme(; fonts=[firasans("Medium"), firasans("Light")])
...
+section of the `Makie.jl` docs.
+
+# Examples
+```jldoctest
+set_aog_theme!()                # Sets a prefedined theme
+update_theme!(fontsize=30,      # Tweaks the current theme
+    markersize=40,
+    Axis = (title="MyDefaultTitle",)
+)
```
Comment (piever) [inline src/theme.jl:149]:
```suggestion
update_theme!(                  # Tweaks the current theme
    fontsize=30,
    markersize=40,
    Axis=(title="MyDefaultTitle",)
)
```

**Review by piever — CHANGES_REQUESTED** [review]

**Comment by piever** [comment]
Thanks for the PR! I've added some comments mostly about wording / style. For CI to pass, the docstrings will need to be included in the documentation website. I imagine the most reasonable place for them would be a `Theming` subsection [here](http://juliaplots.org/AlgebraOfGraphics.jl/dev/API/functions/). They could probably also be referenced [here](http://juliaplots.org/AlgebraOfGraphics.jl/dev/layers/draw/), but that can be done in a separate PR.

**Comment by KronosTheLate** [comment]
Nice! Your suggestions were all improvements in my eyes too, thanks.

It will be some time before I find the time to add a theming section, and I do not have all that much to say about it. Is it something that you would be okay with implementing?

**Review by piever — CHANGES_REQUESTED** [review]

**Review by piever — APPROVED** [review]

**Pull request merged**